crates/notion-core/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "notion-core"
-version = "0.1.0"
+version = "0.2.2"
```

Comment:
Is this important? I've never updated the version numbers of the internal crates since it just seems like busywork. I don't care too much either way but if we're keeping all the internal crates up to date with the main `notion` crate we ought to automate the updating as part of the release process.

Reply:
The reason it's important is that I want to output the Notion version as part of the log file, but `env!("CARGO_PKG_VERSION")` gets the version of the current package. That said, I'm not a huge fan of having to keep the version up-to-date, so I think passing it into the constructor is fine in this case.